man/systemd.path.xml

```diff
@@ -206,6 +206,23 @@
     <xi:include href="systemd.service.xml" xpointer="shared-unit-options" />
   </refsect1>
 
+  <refsect1>
+    <title>Environment Variables Set on Triggered Units</title>
+
+    <variablelist class='environment-variables'>
+
+      <varlistentry>
+        <term><varname>$TRIGGER_REASON</varname></term>
+
+        <listitem><para>The path that caused the unit to be triggered will be set in this environment variable
+        for the associated service unit being triggered. Note that this is provided in a best-effort way.
+        For example, there might be multiple triggers firing at any given time, and only one will be reported,
+        and there is no guarantee on which one it will be.</para></listitem>
+      </varlistentry>
```

Comment:
I think this should be moved to the service man page, as that's where it is set.

Reply:
it already is in systemd.exec - but I think it's good to have it in the trigger manpage, given that's what you look at when using trigger units

Reply:
I think the explanations of the env vars really belongs in systemd.exec and there only. But yes, there hsould be a pointer from the .path + .timer man pages to that.